### ISSUE TITLE
getting Error: All configured authentication methods failed when trying to run ansible playbook remotely via ssh

### ISSUE THREAD

**Issue by ebenezarisaac** [VSChina/vscode-ansible#137]
## Environment
- Extension version: 0.2.6
- OS Type: Windows 7

## Summary
When i am trying to run a simple playbook (.yml file) remotely via SSH, it says "Failed to copy and gave error Error: All configured authentication methods failed". I provided host, ssh password. I also have given servers.json as below with the required details, but it always was giving this error.

[
  {
    "host": "hostname",
    "port": 22,
    "user": "username",
    "password": "password"
    "key": "path to id_rsa"
  }
]

Here is the output in the vscode 

Credential file: C:\Users\m87514\.vscode\ansible-credentials.yml
Copying folder c:\Users\m87514\Documents\vscode-projects\ansible/ to cilsdbxd27328.silver.com.....
Failed to copy c:\Users\m87514\Documents\vscode-projects\ansible/ to cilsdbxd27328.silver.com: Error: All configured authentication methods failed

**Comment by yungezz** [comment]
@ebenezarisaac  thanks for reporting the issue. are your ssh key protected by passphrase? if yes, then it's a bug in existing extension. we have fixing merged pending on release. Stay tuned.

**Comment by yungezz** [comment]
@ebenezarisaac  ssh with passphrase is fixed in v0.3.0, pls have a try and let us know if any fedback!

**Comment by ebenezarisaac** [comment]
Hi, I had set up ssh keys for logging on to remote server . But i also uninstalled ansible extension , then reinstalled and tried to run remotely via ssh. This time it was able to copy the workspace on the remote server. That's fixed. However, i do not see any playbook output in my vscode. Should i be looking for something or is this feature not implemented.

**Comment by yungezz** [comment]
@ebenezarisaac  do you mean no output of ansible-playbook in terminal console?

**Comment by ebenezarisaac** [comment]
Yes i couldnt't see any output that says tasks were executed and changed in the terminal console. It would be helpful to see it while i execute the playbook.

**Comment by yungezz** [comment]
@ebenezarisaac  is there any error message in "outout" window, is there terminal opened with name "SSH <hostname>? could you share a screen cut to help investigate?  thanks.

**Comment by yungezz** [comment]
@ebenezarisaac  the error is from "Yaml Support" extension,  not from this extension. pls see the drop down list in output window, right above. could you pls check "VSCode exntesion for Ansible" to see if there's any error?

**Comment by ebenezarisaac** [comment]
Sorry, my bad. When i check VSCode exntesion for Ansible extension, the output displays the following error.  Looks like im not able to connect to host.

![capture](https://user-images.githubusercontent.com/19736143/43412735-c3362e34-93f3-11e8-94a1-084df6d737f8.PNG)
